__tests__/api/api.test.js

```diff
@@ -0,0 +1,35 @@
+const { seed } = require("../../db/seeds/seed");
+const { app } = require("../../app");
+const request = require("supertest");
+const connection = require("../../db/connection");
+const testData = require("../../db/data/test-data");
+
+
+afterAll(() => {
+    return connection.end();
+});
+
+beforeEach(() => {
+    return seed(testData);
+});
+
+describe('GET /api/topics', () => {
+    test('200: responds with a status of 200', () => {
+      return request(app).get('/api/topics').expect(200);
+    });
+
+    test('200: responds with an array of topics on the body', () => {
+      return request(app)
+        .get('/api/topics')
+        .expect(200)
```

Comment:
Here you are checking status 200 so either this line or the previous test is not really necessary.